i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/deploy-on-kubernetes/overview.md

```diff
@@ -2,61 +2,46 @@
 
 本指南提供了在 Kubernetes 上部署 GreptimeDB 集群的步骤。
 
-## 准备工作
+## 前置条件
 
-在开始部署 GreptimeDB 到 Kubernetes 之前，
-请确保你已经下载了必要的工具，
-准备好 Kubernetes，并创建了命名空间。
+- Kubernetes >= 1.18
 
-### 工具
+  :::tip 注意
+  你可以使用 [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) 或 [Minikube](https://minikube.sigs.k8s.io/docs/start/) 创建一个用于测试的本地 Kubernetes 集群。
+  :::
 
-在部署过程中，除了 [Kubernetes](https://kubernetes.io/) 之外，
-你还需要使用以下工具：
+- [Helm v3](https://helm.sh/docs/intro/install/)：Kubernetes 的包管理器。
 
-- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/)：用于创建 Kubernetes 集群。
-- [Helm](https://helm.sh/docs/intro/install/)：用于 Kubernetes 的包管理器。
 - [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)：用于与 Kubernetes 集群交互的命令行工具。
 
-### 创建 Kubernetes 集群
+## 添加 Helm 仓库
 
-你可以使用 [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) 创建一个用于 GreptimeDB 的 Kubernetes 集群：
+你使用以下命令添加 GreptimeDB Helm chart 仓库。
 
 ```shell
-kind create cluster --name greptime
-```
-
-### 创建 namespace
-
-为了更好地隔离和扩展性，
-建议给 GreptimeDB Operator、etcd 集群和 GreptimeDB 集群使用单独的命名空间。
-
-```shell
-kubectl create namespace greptimedb-admin
-kubectl create namespace etcd
-kubectl create namespace greptimedb-cluster
+helm repo add greptime https://greptimeteam.github.io/helm-charts/
```

Comment:
```suggestion
helm repo add greptime https://downloads.greptime.cn/releases/charts/package
```

Reply:
We don't recommend using the chart again, users can use alicloud oci chart instead.